Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -11,11 +11,10 @@ This guide assumes a Rook cluster as explained in the [Quickstart](../../Getting
 ## Configure an Object Store
 
 Rook has the ability to either deploy an object store in Kubernetes or to connect to an external RGW service.
-Most commonly, the object store will be configured locally by Rook.
-Alternatively, if you have an existing Ceph cluster with Rados Gateways, see the
-[external section](#connect-to-an-external-object-store) to consume it from Rook.
+Most commonly, the object store will be configured in Kubernetes by Rook.
+Alternatively, if you have an existing Ceph cluster with [Rados Gateways](https://docs.ceph.com/en/quincy/radosgw/index.html), see the [external section](#connect-to-an-external-object-store) to consume it from Rook.
```

Comment:
Please adjust based on "you" usage here.

Reply:
fa106d4e19f711f3493b7d8f34791f46dc51e4cc

Reply:
Also, as a preparation, we will ask you to squash all the commits before final merge. It's fine to keep each commit separate while we are working, however.

Reply:
Seems as if this commit has already been squased already. Trying to cherry-pick it I ended up with an empty commit.
If you could please do a quick check whether the "you" based language is not gone.